src/tools/execution-results.h

```diff
@@ -42,13 +42,11 @@ struct LoggingExternalInterface : public ShellExternalInterface {
   Name exportedTable;
   Module& wasm;
 
-  // The name of the imported fuzzing tag for wasm.
-  Name wasmTag;
+  // The imported fuzzing tag for wasm.
```

Comment:
Don't we need to handle an arbitrary number of imported tags?

Reply:
This is just for the special imported tags from JS:

https://github.com/WebAssembly/binaryen/blob/959d522dd31496dc214880739902a022f8cea9ff/scripts/fuzz_shell.js#L386-L391